.github/workflows/nightly_run.yml

```diff
@@ -449,12 +449,6 @@ jobs:
           cmake ..
           cmake --build . --config Release --parallel 4
           ./hello
-      - name: Build Sample [simple]
-        run: |
-          ./build.sh -p host-interp
-          python3 ./sample_test_run.py $(pwd)/out
-          exit $?
-        working-directory: ./samples/simple
```

Comment:
Had better keep build and run simple in nightly run, but needs to update the script. Maybe we can remove it now and add back later.

Reply:
Ok, We add it back later.